src/Elm/Processing.elm

```diff
@@ -225,13 +225,13 @@ findNextSplit dict exps =
         |> Maybe.map (\x -> ( prefix, x, suffix ))
 
 
-highestPrecedence : List ( String, Infix ) -> Dict String Infix
-highestPrecedence input =
+lowestPrecedence : List ( String, Infix ) -> Dict String Infix
+lowestPrecedence input =
     let
         maxi =
             input
                 |> List.map (Tuple.second >> .precedence >> Node.value)
-                |> maximum
+                |> List.minimum
     in
     maxi
```

Comment:
`maxi` seems to be an incorrect variable name in this context. Could you rename `maxi` to `minimumValue`? Or maybe better, just eliminate the `let-in` in the definition of `lowestPrecedence` (`maxi` is only used once as far as I can see).